im-sdk-cli/src/main/java/com/easemob/im/cli/cmd/CreateCmd.java

```diff
@@ -90,12 +112,12 @@ public void contact(@Parameters(index = "0", description = "the user's username"
     @Command(name = "group", description = "Create a group.\n" +
             "Public(by default) groups can be listed using Android, iOS, Web SDKs. So that end user can discover and join public groups.\n" +
             "Private groups can NOT be listed using Android, iOS, Web SDKs.")
-    public void group(@Option(names = "--private", description = "create a private group") boolean isPrivate,
+    public void group(@Parameters(index = "0", arity = "1..*", description = "the member's username list") List<String> members,
                       @Option(names = "--owner", required = true, description = "the owner's username") String owner,
-                      @Option(names = "--member", description = "the member's username") List<String> members,
+                      @Option(names = "--private", description = "create a private group") boolean isPrivate,
                       @Option(names = "--max-members", defaultValue = "200", description = "max number of members") int maxMembers,
-                      @Option(names = "--can-member-invite", defaultValue = "false", description = "can member invite others to join") boolean canMemberInvite,
-                      @Option(names = "--need-approve-to-join", defaultValue = "false", description = "only for public group, whether need approve to join") boolean needApproveToJoin) {
+                      @Option(names = "--can-member-invite", description = "can member invite others to join") boolean canMemberInvite,
+                      @Option(names = {"--need-approve-to-join"}, description = "need approve to join") boolean needApproveToJoin) {
```

Comment:
--need-approve-to-join simply negates --can-member-invite,
so, Let's keep --can-member-invite here.